common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -108,8 +108,12 @@ class CapaFields(object):
     )
     max_attempts = Integer(
         display_name=_("Maximum Attempts"),
-        help=_("Defines the number of times a student can try to answer this problem. "
-               "If the value is not set, infinite attempts are allowed."),
+        help=_(
+            'Defines the number of times a student can try to answer this problem. '
+            'If the value is not set, infinite attempts are allowed. '
+            'NOTE: If a problem is timed, we only allow a single attempt, and ignore '
```

Comment:
@jlikhuva I _think_ this messaging (`problem is timed`) may only be relevant on Stanford's platform;
we have a "timed problem" feature which is different from edX's "timed unit".